Library/Homebrew/tab.rb

```diff
@@ -341,6 +341,20 @@ def write
     tabfile.atomic_write(to_json)
   end
 
+  # Helper method for checking truthiness of a value that may not be specified.
+  # `Tab.might_be?(:foo)` differs from `tab.foo` in that if `foo` is not
+  # specified `#might_be?` will return true.
+  def might_be?(key)
```

Comment:
:+1: to progressing — but the delay here has been on me, not this minor comment! If we can improve it while it's still a work in progress, that's very welcome!

I'm not sure about `key_truthy_or_absent?` though — we're not checking if the key is truthy like that name implies.